setup/context/commands_test.go

```diff
@@ -0,0 +1,69 @@
+package context
+
+import (
+	"fmt"
+	"os"
+	"testing"
+)
+
+type MockFileIO struct {
+	data map[string][]byte
+}
+
+func (mock MockFileIO) ReadFile(path string) ([]byte, error) {
```

Comment:
I would recommend making your method receivers into pointers. This is a pretty good article on why that can be important, aside from consistency with the majority of our other methods: https://nathanleclaire.com/blog/2014/08/09/dont-get-bitten-by-pointer-vs-non-pointer-method-receivers-in-golang/